lib/modules/RESTips.js

```diff
@@ -247,83 +296,47 @@ const tips = [{
 		moduleID: 'userTagger',
 		key: 'vwTooltip',
 	},
-}];
+},
+...Object.values(featureTips),
+];
 
 const initTips = _.once(() => {
 	$('body').on('click', '#disableDailyTipsCheckbox', e => Options.set(module, 'dailyTip', e.target.checked));
-
-	// create the special "you have never visited the console" guider...
-	createGuider(0, 'console');
```

Comment:
Could find anywhere where this was actually run, so I've removed everything relating to `consoleTip`